qemu/tests/block_detect_zeroes.py

```diff
@@ -0,0 +1,108 @@
+"""blockdev detect-zeroes option test"""
+import time
+
+from provider.block_devices_plug import BlockDevicesPlug
+from virttest import error_context, utils_misc, utils_disk, storage, data_dir
+
+from virttest.utils_misc import get_linux_drive_path
+from virttest.utils_windows.drive import get_disk_props_by_serial_number
+
+
+@error_context.context_aware
```

Comment:
No error_context.context() related statement in the following codes, so I think we can remove error_context from line 5 and remove line 11. Thanks.

Reply:
Updated